mmv1/third_party/terraform/tests/resource_datastream_connection_profile_test.go

```diff
@@ -0,0 +1,41 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatastreamConnectionProfile_update(t *testing.T) {
```

Comment:
Are we supposed to be able to update across profile types? For example, removing the gcs_profile block and replacing it with a prosgresql_profile block?

Can you either add a test if this is expected to work, or trigger a ForceNew when one of these is replaced?

Reply:
I'll make the update test do this. Unfortunately, it looks like this will require actually spinning up a postgresql server

Reply:
I'm having some issues getting the connection to postgres to work properly. Looking into it.

Reply:
I am pretty familiar with postgres sql and can assist in spinning up a server if needed. Let me know! I'm free 9-5 MST M-F

Reply:
I've gotten this working, but thanks for the offer :-)